src/lib/swapper/swappers/ThorchainSwapper/utils/addSlippageToMemo.ts

```diff
@@ -2,22 +2,26 @@ import type { ChainId } from '@shapeshiftoss/caip'
 import { BigNumber } from 'lib/bignumber/bignumber'
 import { subtractBasisPointAmount } from 'state/slices/tradeQuoteSlice/utils'
 
-import {
-  DEFAULT_STREAMING_INTERVAL,
-  DEFAULT_STREAMING_NUM_SWAPS,
-  LIMIT_PART_DELIMITER,
-  MEMO_PART_DELIMITER,
-} from './constants'
+import { DEFAULT_STREAMING_NUM_SWAPS, LIMIT_PART_DELIMITER, MEMO_PART_DELIMITER } from './constants'
 import { assertIsValidMemo } from './makeSwapMemo/assertIsValidMemo'
 
-export const addSlippageToMemo = (
-  expectedAmountOutThorBaseUnit: string,
-  quotedMemo: string | undefined,
-  slippageBps: BigNumber.Value,
-  isStreaming: boolean,
-  chainId: ChainId,
-  affiliateBps: string,
-) => {
+export const addSlippageToMemo = ({
+  expectedAmountOutThorBaseUnit,
+  quotedMemo,
+  slippageBps,
+  isStreaming,
+  chainId,
+  affiliateBps,
+  streamingInterval,
+}: {
+  expectedAmountOutThorBaseUnit: string
+  quotedMemo: string | undefined
+  slippageBps: BigNumber.Value
+  chainId: ChainId
+  affiliateBps: string
+  isStreaming: boolean
+  streamingInterval: number | undefined
```

Comment:
Rather than `streamingInterval?: number` for the time being to make sure we pass it everywhere - make me an optional number before opening me

Reply:
Actually there's a better way, to ensure we always pass a `streamingInterval`: 45c716b00fb90fd9073e9444bd801973acd82b1d

Since we always fetch both non/streaming swap quotes next to each other, this will avoid passing `undefined` in the streaming one without the compiler yelling